action/steps/activate.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-if [[ -n "$UNITY_LICENSE" ]]; then
+if [[ -n "$UNITY_LICENSE" || -n "$UNITY_LICENSE_PATH"  ]]; then
```

Comment:
Or operator should be outside of the double square brackets for compatibility reasons

`if [[ -n "$UNITY_LICENSE" ]] || [[ -n "$UNITY_LICENSE_PATH"  ]]; then`